### ISSUE TITLE
scipy: alt text ~~sprint~~ scavenger hunt

### ISSUE THREAD

**Issue by tonyfast** [dask/community#254]
hey there y'all. for scipy this year, we wanted to coordinate an effort to further improve the accessibiity of open source documentation. we already had an event with y'all in #208 that felt fun. we wanted to try a similar format, but over the course of the conference. we were wondering if y'all (or another project in your ecosystem) would be interested in holding an issue open to add some more alt text during scipy. we'll be working on directing folks to those prs, and we are imagining that this might be a useful device in workshops/bofs for some light on boarding. our ultimate goal is to make this a scavenger hunt to add alt-text/contribute to multiple projects during the event.

i'd love to know if any of y'all would be down to coordinate this with myself and @marsbarlee?

**Comment by tonyfast** [comment]
we are fairly certain that this will be inclusive for both in person and virtual. there is still some work to do there, but it is our aim. our biggest challenge will be promoting the virtual part. 

@marsbarlee is going to be there premiering their comic/fanzine about disability awareness in open source. so we are biased on the in person experience because we can advocate better in person. this scavenger hunt portion is a complementary feature of the zine. there will be a digital version of the zine, so it should work. :crossed_fingers:

**Comment by MarsBarLee** [comment]
@jrbourbeau Here's a comic about how this event will run and the role Dask could play in it!
![Alt Text at SciPy Conference](https://user-images.githubusercontent.com/46167686/172898001-7e7a8e81-ab0f-4a20-b312-135a3a5dd9dc.png)

This is a messy draft, so if you have any suggestions or changes, let me know! 

Currently, it looks like we have SciPy and SPEC on board. It would be great to get to more projects on board. That's why there's placeholder logos of a triangle and star. If you know any other project maintainers who might be interested, let us know!

**Comment by jrbourbeau** [comment]
Unfortunately I'm not attending SciPy this year, but am happy to review Dask PRs when then roll in. cc @ncclementi @gjoseph92 @jacobtomlinson @jsignell @jcrist who I know are attending SciPy and @scharlottej13 @ian-r-rose @pavithraes who were involved in the last alt text sprint

**Comment by scharlottej13** [comment]
This is a great idea! I won't be at scipy, but happy to help remotely :). Here's the [PR from our last alt text sprint](https://github.com/scharlottej13/dask/pull/1) for reference.

**Comment by jacobtomlinson** [comment]
Yeah I'll be there an am happy to help. @MarsBarLee the comic looks great, just FYI we recently changed our logo so that will need updating. https://www.dask.org/brand-guide

**Comment by pavithraes** [comment]
I'd love to help remotely too. :)

**Comment by jsignell** [comment]
I'll be there for most of scipy and happy to help. Also I can review/merge PRs after the fact.

**Comment by MarsBarLee** [comment]
> Yeah I'll be there an am happy to help. @MarsBarLee the comic looks great, just FYI we recently changed our logo so that will need updating. https://www.dask.org/brand-guide

@jacobtomlinson Thank you for letting me know about the logo update! It looks great! Once we get the other projects confirmed, I'll clean up the comic, update Dask's logo and post it here.

I'm reading the Dask logo update thread and the video with Matt Rocklin explaining it. I love seeing the evolution and community work when doing design, especially for big visual updates like this.

**Comment by tonyfast** [comment]
aww. its so awesome to see all of the interest. 

we still have a few weeks before the event which means that we shouldn't have to rush too much. we're working on getting a reference pull request together. once we do we will share the example with you, and coordinate with y'all to get `dask` setup for the event.

thanks again for the excitement from y'all. its so great that we can have folks participating in-person and remotely.

**Comment by tonyfast** [comment]
alright y'all we are getting things organized now and thankfully @isabela-pf put together a reference pull request for us.

if you all are still game. here's the best structure we have so far. please let us know if you have any questions.

## how to participate

to participate, we need:

* a project advocate to open a pull request by July 4. _this is the most time sensitive portion so we can print links to your contributions._
  * we generally recommend opening the pull request on a fork to cut down on notifications
  * there is a reference implementation open for [Project Jupyter](#reference), or refer to [past events][past]
* project advocates to add placeholders where alt text is needed in documentation by July 12
* project advocates to communicate _sometimes_ in-person and/or remotely from July 13-14
  * spread word about the project and your pull request
* project advocates to review and close the pull request after July 15
* placeholders provided for participants to discover where alt text needs improvement
  * an example of placeholders for markdown documentation can be found in Jupyter's [reference] pull request.

## why participate

* your project and pull request will be featured in the digital and print scavenger hunt announcement.  we'll add a qr code with your **new** project logo once we have a pull requests to reference. [_see the very rough draft of the scavenger hunt announcements_][draft].
* your project and pull request will be featured in the digital and print in accessibility comic book distributed during SciPy
* it provides a proven way to on-board diverse contributors onto projects
* your documentation will be more accessible 

### reference

there is [a draft pull request started for Project Jupyter][reference], more placeholders will be added. y'all can use this copy to start your pull request, or reference to [past events][past] to find what suits the project best.


[reference]: https://github.com/Quansight-Labs/jupyterlab/pull/1
[past]: https://github.com/isabela-pf/a11y-events/tree/main/workshop-resources/alt-text#past-events
[draft]: https://github.com/alt-text-task-force/.github/blob/main/profile/scipy-2022-07.md
[blog]: https://labs.quansight.org/blog/2021/09/low-code-contributions-through-github/

**Comment by pavithraes** [comment]
@tonyfast Thank you for the details! I've opened a PR here: https://github.com/pavithraes/dask/pull/1

**Comment by tonyfast** [comment]
@pavithraes thank you so much for opening that PR to participate! I added a link with the new logo to our draft manifest for the event. https://github.com/alt-text-task-force/.github/blob/main/profile/scipy-2022-07.md#user-content-fn-history-8189a866323cf2d84748d2fdf2d4ec6b
